chore: removes repetitive wording

ns-rse opened the pull request.

ns-rse committed:
chore: removes repetitive wording

Robadob commented:
CI isn't happy though 🤷‍♂️

ptheywood commented:
Pa11y-ci seems to have become stale, which given it's a nodejs project is probably not a good thing and might be the cause of the issue (i.e. some incompatible dependencies after chromium/puppeteeer updates?)

Someone has forked pa11y-ci and updated dependencies / released newer versions (https://github.com/aarongoldenthal/pa11y-ci / https://www.npmjs.com/package/@aarongoldenthal/pa11y-ci) which might resolve things?

I'll try it in a branch.

ptheywood commented:
Switching to the fork didn't resolve the issue, but did give a slightly more helpful error message from chromium. 

`ubuntu-latest` has changed from being 22.04 to 24.04 between this run last being successful / not being successful (and is mentioned as a warning on the last working run of the workflow 3 weeks ago)

![image](https://github.com/user-attachments/assets/3002fcc3-8bd9-475f-b013-0b0443f3f15a)

Explicitly using `ubuntu-2204` is likely the simplest fix (until something else breaks). I'll make a change in this PR once tested separately.

ptheywood committed:
CI Fix: Use ubuntu-22.04 not ubuntu-latest for pa11y-ci

pa11y-ci uses old versions of chromium & puppeteer which does not agree with the kernel, used in Ubuntu 24.04. Explicitly using the older Ubuntu is the simplest fix (and should be supported until 2027)

ptheywood commented:
CI is now passing on this workflow (but not sure pa11y is actually working & might not have done for a while, but that's a separate problem).

ns-rse commented:
Thanks @ptheywood :+1: 

> `ubuntu-latest` has changed from being 22.04 to 24.04 between this run last being successful / not being successful (and is mentioned as a warning on the last working run of the workflow 3 weeks ago)

Doesn't look like the cause here but R and I don't recall seeing anywhere that uses R markdown but R is no longer bundled in `24.04` and would require explicit installation after spinning up. This might impact the Handbook though after David translated it to Quarto. I'll check and make sure that doesn't break.
